pyanaconda/ui/tui/spokes/installation_source.py

```diff
@@ -102,7 +102,7 @@ def _payload_error(self):
     def _repo_status(self):
         """ Return a string describing repo url or lack of one. """
         if self.data.method.method == "url":
-            return self.data.method.url or self.data.method.mirrorlist
+            return self.data.method.url or self.data.method.mirrorlist or self.data.method.metalink
```

Comment:
Could be better to extract `self.data.method` to a variable `method` and use `method.*` instead everywhere here.

Reply:
Agreed